src/fty-sensor-gpio.conf

```diff
@@ -0,0 +1,2 @@
+# create runtime directories for fty-sensor-gpio
```

Comment:
Runtime RW should be under `/var/lib/fty`. The writes to `/usr` would wear soldered-on flash, writes to `/var` wear out replaceable cards.